src/components/course/enrollment/constants.js

```diff
@@ -0,0 +1,11 @@
+const enrollButtonTypesLocal = {
```

Comment:
these will be used in the logic (so not strictly speaking part of this PR but will be used asap in next work)